docs/products/postgresql/howto/postgresql-debezium-cdc.rst

```diff
@@ -0,0 +1,93 @@
+Handle PostgreSQL® node replacements when using Debezium for change data capture
+=================================================================================
+
+As an Aiven customer you may be running a Aiven for Apache Kafka® Connect cluster that runs one or more `Debezium PostgreSQL® source connectors <https://docs.aiven.io/docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.html>`__ (Debezium) capturing changes from an Aiven for PostgreSQL® service.
+
+After the PostgreSQL service has had a maintenance update or undergo any operation which replaces the nodes (such as a plan or cloud region change), Debezium would lose connection to the PostgreSQL database. It is able to recover from connection errors to the database and start listening for change events again after a restart of the connector's task. However, in some cases the Debezium replication slot is lagging behind as the amount of WAL logs grows causing a concern.
+
+.. Tip::
+   
+   There is a GitHub repository where you can easily spin up a Debezium - PostgreSQL setup to test the node replacement scenario and also follow along this `Aiven Debezium help article <https://github.com/aiven/debezium-pg-kafka-connect-test>`__.
+
+The possible errors encountered, the cause and solution to the growing replication slot lag, and how to ensure a graceful PostgreSQL node replacement or maintenance upgrade are explained in the next sections.
+
+
+Debezium errors related to PostgreSQL node replacement
+------------------------------------------------------------------------------
+
+During the reported PostgreSQL node replacements where Debezium could not recover, we have observed the following errors that have occurred in one or more occasions::
+
+   # ERROR 1
+   org.apache.kafka.connect.errors.ConnectException: Could not create PostgreSQL connection
+   # ERROR 2
+   io.debezium.DebeziumException: Could not execute heartbeat action (Error: 57P01)
+   # ERROR 3
+   org.PostgreSQL.util.PSQLException: ERROR: replication slot "SLOT_NAME" is active for PID xxxx
+
+These are all unrecoverable errors, meaning that they require a restart of the connector task(s) in order to resume operations again. A restart can be performed manually either through the `Aiven Console <https://console.aiven.io/>`_, in under the `Connectors` tab console or via the `Kafka Connect REST API <https://docs.confluent.io/platform/current/connect/references/restapi.html#rest-api-task-restart>`__. You can get the service URI from the `Aiven Console <https://console.aiven.io/>`_, in the service detail page.
+
+.. image:: /images/products/postgresql/pg-debezium-cdc_image1.png
+   :alt: The Aiven Console page showing the Debezium connector error
```

Comment:
@zhangmmaiven I believe the error was related to the missing `:alt:` for the image